playlistmanager.lua

```diff
@@ -635,8 +635,29 @@ function playlist(force_dir)
   if playlist_visible then showplaylist() end
 end
 
+local function check_playlist_savapath()
+  if settings.playlist_savepath == nil or utils.readdir(settings.playlist_savepath) == nil then
+		
+    local default_playlist_savepath = mp.command_native({"expand-path", "~~home/"}).."/playlists"
+    msg.info("The custom playlist save path is invalid. Use the default path: "..default_playlist_savepath)
+		
+    if mp.find_config_file("playlists") then
+      settings.playlist_savepath=mp.find_config_file("playlists")
+    else
+      msg.info("Default playlist save path does not exist. It will be created automatically.")
+      if os.getenv("windir") then
```

Comment:
I think we should use subprocess here instead. Mainly because it will escape paths with special characters and spaces.

```
windows_args = {'powershell', '-NoProfile', '-Command', 'mkdir', default_playlist_savepath}
unix_args = { 'mkdir', default_playlist_savepath }
```

Reply:
Use the `settings.system` to check system for consistency.